sites/platform/src/dedicated-environments/backups-restores.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Dedicated backup and restores"
+weight: 1
+sidebarTitle: "Dedicated backups"
+layout: single
+description:  "Backups are retained for different periods depending on various factors and whether you’re using a Dedicated Gen 2 or Dedicated Gen 3 Environment. These processes can be either manual or automated."
+---
+
+{{% description %}}
+
+
+## Dedicated Generation 2 Backups
+
+Platform.sh takes a byte-for-byte snapshot of Dedicated Gen 2 production Environments every 6 hours. Backups are retained for different durations depending on when they were taken. 
```

Comment:
@gilzow you had reviewed that the "when they were taken" phrasing in the table wasn't the best. What do you think of this? https://4221-improvements-to-jluuz2y-ucq44jg6ofare.eu-5.platformsh.site/dedicated-environments/backups-restores.html#dedicated-generation-2-backups